less/common/Checkbox.less

```diff
@@ -3,8 +3,9 @@
   cursor: pointer;
   margin: 0;
 
-  & input[type="checkbox"] {
-    display: none;
+  input[type="checkbox"] {
```

Comment:
Why not &?

Reply:
it doesn't make a difference, it's the same.